pulp_container/app/content.py

```diff
@@ -6,8 +6,6 @@
 
 registry = Registry()
 
-app.add_routes([web.get('/pulp/container/', registry.serve_v2)])
-app.add_routes([web.get('/pulp/container/_catalog', registry.list_repositories)])
```

Comment:
@dkliban  i think we can also move tags/list to the urls.py

Reply:
i agree